SmartDeviceLink/SDLMenuManager.m

```diff
@@ -274,6 +418,17 @@ - (void)sdl_sendCurrentMenu:(SDLMenuUpdateCompletionHandler)completionHandler {
 
 #pragma mark - Helpers
 
+- (BOOL)sdl_isDynamicMenuUpdateActive:(SDLDynamicMenuUpdatesMode)dynamicMenuUpdatesMode {
+    switch (dynamicMenuUpdatesMode) {
+        case SDLDynamicMenuUpdatesModeForceOn:
+            return true;
+        case SDLDynamicMenuUpdatesModeForceOff:
+            return false;
+        case SDLDynamicMenuUpdatesModeOnWithCompatibility:
+          return ![self.displayCapabilities.displayName isEqualToEnum:SDLDisplayTypeGen38Inch];
```

Comment:
This is currently broken

```suggestion
          #pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
          return ![self.displayCapabilities.displayType isEqualToEnum:SDLDisplayTypeGen38Inch];
#pragma clang diagnostic pop
```

Reply:
If I make a Sync 3 menu cell that changes the normal example app menu to only have the submenu menu, it will fail every submenu `addCommand` even after the above change.

Reply:
> If I make a Sync 3 menu cell that changes the normal example app menu to only have the submenu menu, it will fail every submenu `addCommand` even after the above change.
This issue is fixed. I was using the incorrect reference to index.

Reply:
See the first comment on this thread, this is still broken.

Reply:
fixed this issue!